src/Forms/SingleLookupField.php

```diff
@@ -94,7 +94,7 @@ public function Value()
             return $label;
         }
 
-        return $value;
+        return $this->value;
```

Comment:
How about this?

```suggestion
        return parent::Value();
```

Reply:
Good point. I just implemented your suggestion.